lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -433,6 +459,67 @@ describe('BasicLtiLaunchApp', () => {
       await waitForElement(wrapper, 'FakeDialog[title="Authorize Hypothesis"]');
       await contentHidden(wrapper);
     });
+
+    context('when auth fails multiple times in a row', () => {
+      // Helper to ensure the dialog is initially present but also remains so
+      // for a short duration longer without disappearing.
+      async function dialogShouldRemain(wrapper) {
+        // Error prompt should be present.
+        assert.isTrue(wrapper.find('FakeDialog').exists());
+        // Error prompt should also not go away after a short while (e.g.flicker)
```

Comment:
I suggest to replace `(e.g. flicker)` with `(see issue #1826)` which describes the issue that this test intends to guard against in more detail.